easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.20-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,110 @@
+    ('seqtk', '1.2-r101c', {
+        'easyblock': 'MakeCp',
+        'source_urls': ['http://github.com/ndaniel/seqtk/archive'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['4a57fbe828eccaa4f0122c84bf9c4c5409a024ad9deace31ec7bee6d25cf5e49'],
+        'start_dir': '%(name)s-%(version)s',
+        'files_to_copy': [(['seqtk'], 'tools/%(name)s')],
+    }),
+    # FusionCatcher requires this very specific version of BBMap
+    ('BBMap', '38.44', {
+        'easyblock': 'MakeCp',
+        'source_urls': [SOURCEFORGE_SOURCE],
+        'sources': ['%(name)s_%(version)s.tar.gz'],
+        'checksums': ['63ecdc83cdb0aaf5106b919df487f50a6b65e816d990501e9f74704c04b2a84a'],
+        'start_dir': '%(namelower)s',
+        'prebuildopts': "cd jni && ",
+        'buildopts': "-f %s" % local_bbmap_makefile,
+        'files_to_copy': [(['*'], 'tools/%(namelower)s')],
+    }),
+]
+
+# Set paths in the configuration file
+postinstallcmds = [
+    'sed -i "s|python = /usr/bin/|python = $EBROOTPYTHON/bin/|g" %(installdir)s/etc/configuration.cfg',
+    'sed -i "s|java = /usr/bin/|python = $EBROOTJAVA/bin/|g" %(installdir)s/etc/configuration.cfg',
+    'sed -i "s|paralell/src|parallel/bin|g" %(installdir)s/etc/configuration.cfg',
+]
+
+# Create symlinks in installation dir to dependencies from EB
+postinstallcmds += [
+    'ln -s $EBROOT%s %%(installdir)s/tools/%s' % (d, d.lower()) for d in [
+        'BLAT', 'BOWTIE', 'BOWTIE2', 'BWA', 'PARALLEL', 'PICARD', 'PIGZ', 'STAR', 'VELVET'
+    ]
+]
+postinstallcmds += [
+    'ln -s $EBROOT%s %%(installdir)s/tools/%s' % d for d in [
+        ('KENT_TOOLS/bin', 'liftover'),
+        ('SRAMINTOOLKIT', 'sratoolkit'),
+        ('BIOPYTHON/lib/python%(pyshortver)s/site-packages/Bio', 'biopython'),
+        ('SCIPYMINBUNDLE/lib/python%(pyshortver)s/site-packages/numpy', 'numpy'),
+        ('OPENPYXL/lib/python%(pyshortver)s/site-packages/openpyxl', 'openpyxl'),
+        ('PYTHON/lib/python%(pyshortver)s/site-packages/xlrd', 'xlrd'),
+    ]
+]
+
+# Link the required Ensembl database
+# Database can be created after install with `fusioncatcher-build -g homo_sapiens -o /path/to/human_v98`
+# More information at https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md
+# postinstallcmds += [
+#     'ln -s /path/to/database/human_v98 %(installdir)s/data/human_v98',
+#     'cd %(installdir)s/data && ln -s human_v98 current',
+# ]
+
+# Tests need 16 cores and 64 GB of RAM as well as the Ensembl database
+# postinstallcmds += ['%(installdir)s/test/test.sh']
+
+sanity_check_paths = {
+    'files': ['bin/fusioncatcher', 'bin/fusioncatcher-batch', 'bin/fusioncatcher-build', 'etc/configuration.cfg',
+              'tools/seqtk/seqtk', 'tools/bbmap/bbmap.sh', 'tools/bbmap/jni/libbbtoolsjni.%s' % SHLIB_EXT],
+    'dirs': ['data', 'test'],
+}
+
+sanity_check_commands = ["fusioncatcher --help"]
+
```

Comment:
I've included this modloadmsg:
```
modloadmsg = "FusionCatcher needs database to work properly.\n"
modloadmsg += "Database download script is located in: ${EBROOTFUSIONCATCHER}/data/download-human-db.sh\n"
modloadmsg += "Path to downloaded data folder can be set with '--data' option in fusioncatcher"
```
Not sure why `fusioncatcher-build` is needed. 
According to doc: https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md#413---installation-from-github manual instalation uses `download-human-db.sh`

The download-human-db.sh just downloads 4 part tar.gz which `cat` together and unpack into folder. It doesn't even include `-c` continue flag for those 4 files each having ~ 4GB. If you hit slow mirror you will wait a day to dl one part...

Reply:
There are two options, one is to download the database as you just did with `download-human-db.sh `, the other one is two download a much smaller tarball and build the database on-site. `fusioncatcher-build` is needed for the latter.

Reply:
I see :) than `fusioncatcher-build` might be better option :+1:

Reply:
I have rewritten the comments describing the installation of the database with the information included in your `modloadmsg`, including both options the download and the built on-site. I prefer to not add any `modloadmsg` to this easyconfig though, as that is something very site-specific. Let me know if I missed anything.

Reply:
Althought I thought that `modloadmsg` is for scientist who uses the module, since I doubt they will dig through easyblock comments, which was the reason I've included the info there in first place.

Reply:
Absolutely, those comments target the admins. How the databases are handled can vary a lot from site to site. For instance, in our site we (the admins) install the databases for our users and hence we only provide the path to those databases. Thus, I added a `modloadmsg` to this easyconfig telling our users the exact location of the `human_v98` database in our system and nothing else. Other sites might do other things.

Reply:
I see :) That makes sense, thanks for explanation.